File: java/assign4/src/CurvedBanner.java
package BannerPrice;

public class CurvedBanner extends Banner{

		private float radius;

//	public CurvedBanner(){
		
//		super();

//		radius = 0.5f;
//	}

	public CurvedBanner(int h , int w , float r){
	
		super(h,w);

		radius = r;
	}

	public float getRadius(){
	
		return radius;
	}

	public void setRadius(float r){
	
		radius = r;
	}

	public double area(){
		
		return super.area()- 0.86 * radius * radius;
	}
}

File: dotnet/assignment3/DemoApp/CurvedBanner.cs
namespace bannerPrice;

public class CurvedBanner : Banner{

private float radius {get; set;}

public CurvedBanner(int h , int w , int r) : base( h , w)
{
    radius = r;
    

}

/*public float Radius{
    get { return radius;}
    set { radius = value;}
}
*/
public override double area(){
    return base.area()- 0.86 * radius * radius;
	}
}

